### PR TITLE
[excel] (Shapes) Adding shape moving and ordering snippet

### PR DESCRIPTION
**Pull request by AlexJerabek**
This snippet shows how to move, resize, rotate, and reorder shapes in a worksheet.

### CODE REVIEW

File: playlists/excel.yaml [hunk 1]
```diff
@@ -629,14 +629,14 @@
   group: Worksheet
   api_set:
     ExcelApi: 1.1
-- id: excel-shapes-create-and-delete
+- id: excel-shape-create-and-delete
```
Comment (ElizabethSamuel-MSFT) [inline playlists/excel.yaml]: curious about reason for rename...?

Comment (AlexJerabek) [reply playlists/excel.yaml]: I wanted to be consistent with "range" and "worksheet".

File: samples/excel/85-preview-apis/shape-move-and-order.yaml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                <span class="ms-Button-label">Move square left</span>
+            </button><p/>
+            <button id="moveDown" class="ms-Button">
+                <span class="ms-Button-label">Move pentagon down</span>
+            </button><p/>
+            <button id="rotate" class="ms-Button">
+                <span class="ms-Button-label">Rotate the pentagon</span>
+            </button><p/>
+            <button id="scaleUp" class="ms-Button">
+                <span class="ms-Button-label">Resize the octogon</span>
```
Comment (ElizabethSamuel-MSFT) [inline samples/excel/85-preview-apis/shape-move-and-order.yaml]: typo: should be octagon

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT Thanks for the review. The typos have been fixed.

**Pull request merged**